### ISSUE TITLE
Change "amino-acid" to "amino acid"

### ISSUE THREAD

**Issue by cmungall** [ebi-chebi/ChEBI#4527]
CHEBI uses a mixture of amino-acid and amino acid. I think the latter is correct.

This seems trivial but it's often necessary to parse CHEBI terms because conjugate based edges are incomplete, and the need to normalize hyphenation can complicate this process

I suspect this is why there are no conjugate base/acid relationships between "amino acid" and "amino-acid anion" in CHEBI

**Comment by amalik01** [comment]
@cmungall I agree that it does become confusing at times but this is the way IUPAC recommends it. When the phrase 'amino acid' is a qualified noun it contains no hyphen; a hyphen is inserted when it becomes an adjective so as to join its components in qualifying another noun, e.g. amino-acid sequence (see https://iupac.qmul.ac.uk/AminoAcid/AA1n2.html).

**Comment by amalik01** [comment]
another example is amino-acid residue (https://goldbook.iupac.org/terms/view/A00279)
